modules/twinkleblock.js

```diff
@@ -433,9 +433,37 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 			tags: true,
 			placeholder: 'Enter pages to block user from',
 			maximumSelectionLength: 10, // Software limitation [[phab:T202776]]
-			language: {
-				noResults: function() {
-					return 'No pages entered yet';
+			minimumInputLength: 1, // prevent ajax call when empty
+			createTag: function() { // prevent selection of arbitrary titles not returned by API
```

Comment:
Dunno what happened. I was trying to convert my [StubSorter](https://en.wikipedia.org/wiki/User:SD0001/StubSorter.js) to use select2 (currently implemented in chosen with One Big Hack for ajax lookups), but the ajax thing wasn't working at all. I left it. Later, on trying out the same here, it worked, and I had assumed that it was because of `tags: true` which I hadn't put in StubSorter. 

But you're right. It works even without it. `tags: true` and `createTag: function() { return null; }` appear to be cancelling each other out. Both appear to be unnecessary.

Reply:
`tags` is just for user-generated input, and would be useful if nonexistent pages were allowed (hence, I mistakenly used it initially) but yeah, no need here.